README.md

```diff
@@ -768,6 +769,92 @@ loggingClient.getEntries(function(err, entries) {
 ```
 
 
+## Google Cloud Natural Language (Beta)
+
+> **This is a Beta release of Google Cloud Natural Language.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-language-docs]
+- [Official Documentation][cloud-language-docs]
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var language = gcloud.language;
+```
+
+#### Using the Natural Language API module
+
+```
+$ npm install --save @google-cloud/language
+```
+
+```js
+var language = require('@google-cloud/language');
+```
+
+#### Preview
+
+```js
+var gcloud = require('google-cloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var languageClient = language({
```

Comment:
Oh woops, I have to remove the `var gcloud = require('google-cloud');` line. The `language` variable is created in two different ways above the API example.